### ISSUE TITLE
4 in 1 multi protocol, Taranis x9e and opentx 2.19, hubsan protocol and telemetry

### ISSUE THREAD

**Issue by dprerry20** [pascallanger/DIY-Multiprotocol-TX-Module#50]
I installed the 4 in 1 module in my taranis x9e. Upgraded firmware on transmitter and modified the 4in1 muti protocol board from bangood version 1.2. 
I can get the transmitter to bind with the hubsan h502s
I have no telemetry even though it was enabled in the config.h file. 
The only way I can get the quad to arm is by taking it outside till it finds at least 6 satelites.
I have no way of over riding the fly with no gps.
Any way of being able to get these function on the ex9

Thank you guys for all your work

**Comment by pascallanger** [comment]
Hubsan protocol does not support all this yet. This is a future improvement.

**Comment by dprerry20** [comment]
Thank you

------ Original Message ------

**Comment by pascallanger** [comment]
I'll look at the pull request for hubsan when I'm be back from vacation. Stay tune.
Pascal

**Comment by dperry20** [comment]
I tried the code yesterday. It didn't do anything. I am not a 
programmer. I did some programing in fortran and basic in my earlier 
days. There were a couple of errors which I think I fixed because when I 
ran the sketch, no further errors showed up.

I uploaded the sketch to my 4 in 1. Nothing showed up. Is there anyone 
that can check the code that I uploaded for the Hubsan.

Danny

------ Original Message ------
